chevah_build

```diff
@@ -180,13 +187,21 @@ case $OS in
             # We favour the BSD-flavoured "install" over the default one.
             # "ar", "nm" and "ld" are included by default in the same path.
             export PATH=/usr/ccs/bin/:$PATH
-            # sqlite3 lib location in all Solaris'es (incl. 10s10 for Sparc).
+            # sqlite3 lib location in all Solaris'es (incl. 10s10 for Sparc)
+            # is /usr/lib/mps.
+            # Solaris 10u3 does not come with sqlite3 or gmp, and zlib is
+            # outdated and breaks the build. If we manually install them,
```

Comment:
why should we **manually** install them.

the whole purpose of the python-package and this chevah_build script is to automate the whole build process